packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -1066,20 +1000,57 @@ export class StatsAnalyzer extends EventsScope {
       if (currentPacketsLost < 0) {
         currentPacketsLost = 0;
       }
-
       const currentPacketsReceived =
         result.packetsReceived - this.statsResults[mediaType][sendrecvType].totalPacketsReceived;
       this.statsResults[mediaType][sendrecvType].totalPacketsReceived = result.packetsReceived;
 
       if (currentPacketsReceived === 0) {
         if (receiveSlot && sourceState === 'live') {
           LoggerProxy.logger.info(
-            `StatsAnalyzer:index#processInboundRTPResult --> No packets received for receive slot ${idAndCsi}. Total packets received on slot: `,
+            `StatsAnalyzer:index#processInboundRTPResult --> No packets received for mediaType: ${mediaType}, receive slot ${idAndCsi}. Total packets received on slot: `,
             result.packetsReceived
           );
         }
       }
 
+      if (mediaType.startsWith('video') || mediaType.startsWith('share')) {
+        const currentVideoFramesReceived =
```

Comment:
nitpick: "current" in that name maybe a bit confusing, maybe better to call it `videoFramesReceivedDiff`? (same for frames decoded and dropped)